app/services/call_creator.rb

```diff
@@ -46,10 +46,16 @@ def errors
   private
 
   def sanitize_params!
-    if @params[:member_phone_number].present?
-      @params[:member_phone_number] = Phony.normalize(@params[:member_phone_number])
+    begin
+      if @params[:member_phone_number].present?
+        @params[:member_phone_number] = Phony.normalize(@params[:member_phone_number])
+      end
+    rescue Phony::NormalizationError
+    end
+
+    if @params[:member_id].blank? && @extra_params[:akid].present?
+      @params[:member_id] = Member.find_by_akid(@extra_params[:akid])&.id
```

Comment:
Yeah, I agree consistency would be great. When I first wrote this like this this was my proposal to start using this new format, but I definitely missed that `recognized_member` would fail at the time. I also learnt later that AK mailings doesn't support nested params. All this points out that maybe the easiest thing is to fall back to the root level params for everything.